index.html

```diff
@@ -0,0 +1,1681 @@
+                                    <picture class="mapping-sm_md lazysize">
+                                        <!--[if IE 9]><video style="display: none;"><![endif]-->
+                                        <source data-srcset="https://s.newsweek.com/sites/www.newsweek.com/files/styles/sm/public/2019/01/26/meghan-mccain.jpg 1x"
+                                            media="(min-width: 992px)" srcset="https://s.newsweek.com/sites/www.newsweek.com/files/styles/sm/public/2019/01/26/meghan-mccain.jpg 1x">
+                                        <source data-srcset="https://s.newsweek.com/sites/www.newsweek.com/files/styles/sm/public/2019/01/26/meghan-mccain.jpg 1x"
+                                            media="(min-width: 768px)" srcset="https://s.newsweek.com/sites/www.newsweek.com/files/styles/sm/public/2019/01/26/meghan-mccain.jpg 1x">
+                                        <source data-srcset="https://s.newsweek.com/sites/www.newsweek.com/files/styles/sm/public/2019/01/26/meghan-mccain.jpg 1x"
+                                            media="(min-width: 481px)" srcset="https://s.newsweek.com/sites/www.newsweek.com/files/styles/sm/public/2019/01/26/meghan-mccain.jpg 1x">
+                                        <source data-srcset="https://s.newsweek.com/sites/www.newsweek.com/files/styles/sm/public/2019/01/26/meghan-mccain.jpg 1x"
+                                            media="(min-width: 0px)" srcset="https://s.newsweek.com/sites/www.newsweek.com/files/styles/sm/public/2019/01/26/meghan-mccain.jpg 1x">
+                                        <!--[if IE 9]></video><![endif]-->
+                                        <img  class="mapping-sm_md lazysize featured-photo  lazyloaded" src="https://s.newsweek.com/sites/www.newsweek.com/files/styles/sm/public/2019/01/26/meghan-mccain.jpg"
+                                            alt="GettyImages-1085827764" title=""></picture>
+                                </a></div>
+                        </div>
+                        <h3><a href="#">Meghan McCain No Longer Calls Herself A Republican</a>
+                        </h3>
+                        <div class="summary">“I honestly don’t believe [Trump and Pence] intended to do anything for
+                            Dr. King’s birthday," Rev. Al Sharpton said.</div>
+                        <ul class="related">
+                            <li>
+                                <a href="/insult-american-people-al-sharpton-trump-mlk-day-events-1299344">Chris Cuomo,
+                                    Meghan McCain Tackle Journalism, Acosta</a></li>
+                        </ul>
+                    </article>
+
+                    <article>
+                        <div>
+                            <div class="category"> <a href="/world">World</a></div>
+                            <div class="image"> <a href="/migrants-refugees-exotic-diseases-world-health-organization-1299685">
```

Comment:
You have a lot of weird references on your `a` tags. Did you copy and paste HTML elements from the original site?